packages/contracts/contracts/foundry/Sphinx.sol

```diff
@@ -266,6 +266,10 @@ abstract contract Sphinx {
         return sphinxUtils.getGnosisSafeProxyAddress(sphinxConfig);
     }
 
+    function sphinxLibraryVersion() public view returns (string memory) {
+        return sphinxUtils.getSphinxLibraryVersion();
+    }
```

Comment:
Thoughts on adding the library version to the `deploymentInfo` struct that we create in the `sphinxCollect` function in `Sphinx.sol`? This way we don't need to make another `forge script` call in our CLI code, which would be nice because forge script calls take a few seconds each

Reply:
That's basically what I'm planning to do, but it'll come in a separate PR when I implement the logic for checking if the contracts are out of date.

Reply:
Sounds good. When you do that, I think it'd be best to remove this function from `Sphinx.sol` to minimize the number of user-facing functions. We should be able to just call `sphinxUtils.getSphinxLibraryVersion()` in the `sphinxCollect` function